src/main/java/org/traccar/protocol/RuptelaProtocolDecoder.java

```diff
@@ -116,6 +122,9 @@ private void decodeParameter(Position position, int id, ByteBuf buf, int length)
             case 80:
                 position.set(Position.PREFIX_TEMP + (id - 78), readValue(buf, length, true) * 0.1);
                 break;
+            case 95:
+                position.set(Position.KEY_OBD_SPEED, UnitsConverter.knotsFromKph(readValue(buf, length, true)));
```

Comment:
OBD speed we actually store in km/h usually, from what I remember.

Reply:
I went with this comment:  https://github.com/traccar/traccar/blob/7f2ded68ef015684f90bef59c055a1c980688069/src/main/java/org/traccar/model/Position.java#L98

Also this decoder does similar conversion:
https://github.com/traccar/traccar/blob/7f2ded68ef015684f90bef59c055a1c980688069/src/main/java/org/traccar/protocol/Gl200TextProtocolDecoder.java#L662

On the other hand, TotemProtocolDecoder is storing raw value, which is km/h per my documentation
https://github.com/traccar/traccar/blob/7f2ded68ef015684f90bef59c055a1c980688069/src/main/java/org/traccar/protocol/TotemProtocolDecoder.java#L549

I would prefer km/h storing too - but then we need to update the comment in Position.java and fix at least the Gl200TextProtocolDecoder. Should I do it?

Reply:
hmmm.. I guess most of the places use km/h, so maybe update those inconsistent places.